packages/nvidia_nat_mcp/src/nat/plugins/mcp/auth/auth_flow_handler.py

```
@@ -142,3 +151,29 @@ async def _handle_oauth2_auth_code_flow(self, cfg: OAuth2AuthCodeFlowProviderCon
+            if not self._redirect_app:
+                raise RuntimeError("Redirect app not built.")
+
+            self._server_controller = _FastApiFrontEndController(self._redirect_app)
+
+            asyncio.create_task(self._server_controller.start_server(host=self._redirect_host,
+                                                                     port=self._redirect_port))
+            logger.debug("MCP redirect server starting on %s:%d", self._redirect_host, self._redirect_port)
+
+            # Give the server a moment to bind sockets before we return
+            await asyncio.sleep(0.3)
+        except Exception as exc:  # noqa: BLE001
```

Ruff (0.14.0)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)